beeline/trace.py

```diff
@@ -375,3 +388,17 @@ def inner(*args, **kwargs):
                     return fn(*args, **kwargs)
             return inner
     return wrapped
+
+
+# Use system random instead of default psuedorandom generator
+system_random = random.SystemRandom()
+
+
+def generate_span_id():
+    """Generate span ID compatible with w3c tracing spec."""
+    return "{:016x}".format(system_random.getrandbits(64))
```

Comment:
the other beelines have pulled the lengths out to constants (although they're expressed in bytes not bits). examples: [node](https://github.com/honeycombio/beeline-nodejs/pull/216/files#diff-d08849b7348979d0e5a64f5eda126a2eR22-R24), [go](https://github.com/honeycombio/beeline-go/pull/110/files#diff-c74a48081d4d8360b4a570379a806b79R16-R19), [java](https://github.com/honeycombio/beeline-java/pull/50/files#diff-6ad2759aa212fe9db3b8c426bc03fd72R15-R16)